components/raftstore-v2/src/operation/mod.rs

```diff
@@ -7,4 +7,5 @@ mod ready;
 
 pub use command::{SimpleWriteDecoder, SimpleWriteEncoder};
 pub use life::DestroyProgress;
+pub use query::local::{CachedReadDelegate, LocalReader, MsgRouter, StoreMetaDelegate};
```

Comment:
Why export `CachedReadDelegate`, `MsgRouter` and `StoreMetaDelegate`?

Reply:
StoreMetaDelegate is needed while the others are not now.